src/main/java/io/dockstore/provision/SynapsePlugin.java

```diff
@@ -77,10 +68,8 @@ public void setConfiguration(Map<String, String> map) {
         public boolean downloadFrom(String sourcePath, Path destination) {
             SynapseClient synapseClient = new SynapseClientImpl();
             try {
-                String synapseKey = config.get("synapse-api-key");
-                String synapseUserName = config.get("synapse-user-name");
-                synapseClient.setApiKey(synapseKey);
-                synapseClient.setUsername(synapseUserName);
+                String synapsePersonalAccessToken = config.get("synapse-personal-access-token");
```

Comment:
Probably overkill, and totally optional, especially since I don't think we get much usage on this plugin, but you could:

1. If synapse-api-key and/or synapse-user-name are present, print a warning that they are no longer used
2. If synapse-personal-access-token is not set, throw an error. Or at don't call line 72.

In case anybody is using this with the old params and things start breaking....